app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = current_user.recipes
+  end
+
+  def show
```

Comment:
- [ ] Update options for recipes is missing! Recipes should be able to be either _updated_ or _deleted_. Kindly ensure to  improve upon this aspect.

**Your design:**        |  **Expected:**
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/12745474/207636441-c66a3f6f-03bc-446b-954d-b0980426f7ce.png) |  ![image](https://user-images.githubusercontent.com/12745474/207636828-067e0902-f4d7-40da-9131-fee58a99712e.png)

Reply:
Done